acapy/start.sh

```diff
@@ -2,9 +2,12 @@
 
 set -euxo pipefail
 
-MEDIATOR_URL=$(curl -s http://ngrok:4040/api/tunnels | jq -r '[.tunnels[] | select(.proto == "https")][0].public_url')
+if [ -z  "${MEDIATOR_URL-}" ];
+then
+    MEDIATOR_URL=$(curl -s http://ngrok:4040/api/tunnels | jq -r '[.tunnels[] | select(.proto == "https")][0].public_url')
+fi
 
-echo "Starting agent with endpoint(s): ${MEDIATOR_URL} wss://${MEDIATOR_URL#*://*}"
+echo "Starting agent with endpoint(s): ${MEDIATOR_URL} ws${MEDIATOR_URL/http/}"
```

Comment:
I think this is often created as `wss://foo.com/ws`. Can we use `ws` as the trailing path without the final slash. If you find this is required then sure - we can roll with it.